jerry-libc/include/assert.h

```diff
@@ -25,6 +25,11 @@ extern "C"
 #endif /* __cplusplus */
 
 #ifndef NDEBUG
+
+#ifndef __GNUC__
+#define __builtin_expect(expression, expected_value) (expression)
+#endif /* !__GNUC__ */
```

Comment:
Guys -- I thought the only reason this hadn't yet been merged was b/c I forgot to ping Tillman that I had added the DCO message (sorry, Tillman! :-)  -- what am I missing/what can I do to fix this other problem?  (I had assumed that if the code passed Travis that it was good to go...?)